server/manual-rendering.md

```diff
@@ -0,0 +1,63 @@
+### Manual rendering
```

Comment:
I agree we should have that. I think we have it [here](https://github.com/dev-academy-programme/concepts/blob/master/server/templates.md), which I think is included on Monday, week 2.